### ISSUE TITLE
0.16.0 Discussion

### ISSUE THREAD

**Issue by orphu** [orphu/mcdungeon#355]
Looks like 1.10 will be out right soon. Moving the milestone to just be 0.16.0 for MC 1.10.

**Comment by JiFish** [comment]
Really not much to add for this version.

Looks like there's been some changes to mob tags, so we best check the spawners work. We can now also create skeletons that fire effect arrows. Interesting possibilities!

There's also the new stray and husk mob variants. These use the SkeletonType/ZombieType tag so they'll have to be customised spawners. It would be nice to move the mob spawning stuff away from NBT files.

1.10 change list: http://minecraft.gamepedia.com/1.10

**Comment by orphu** [comment]
Well, that was fast. 

http://www.mojang.com/2016/06/minecraft-110-an-update-of-fire-and-ice/

**Comment by JiFish** [comment]
Other than #357 everything targeted for 0.16.0 is done.

**Comment by orphu** [comment]
You're probably right. I've not had much opportunity to test.

**Comment by galac17** [comment]
Do you have a target date for the 0.16.0 release?
I use the pre-built packages with the interactive launcher.bat.
I would love to try out the "incomplete" chunks fix.

I have this issue with the new 1.10 world I generated with WorldBorder and it does not look like a WorldBorder fix for the "populating" of the blocks is coming anytime soon.

Thanks for a great add-on...my  players like the dungeons.

**Comment by JiFish** [comment]
Hi @galac17 

I'm afraid there is no timescale beyond "When @orphu has a chance to test and compile the new version." The Milestone target date has passed, but that's just a guideline not a deadline.

The development version is pretty stable though. If you wanted to try it out, you can get it from here: https://github.com/orphu/mcdungeon/archive/development.zip

You'll need to install Python 2.7 and Numpy to get it to run. Once they are installed and the download is unzipped, go to the directory in a command prompt and type:
`python mcdungeon.py interactive`
to enter the interactive mode.

**Comment by orphu** [comment]
Sorry for the delay folks!

I think I'll have time to merge into main and build a binary release this weekend.

**Comment by galac17** [comment]
Thanks...I went ahead and loaded Python an Numpy (fun one) to my Minecraft
server and then pulled all the files and the couple of other dependencies.
I am not real familiar with Python or pulling code from GitHub...but I got
it sorted out.

I was able to run the new code with the Incomplete chunk bypass and it
worked nicely as far as I can tell.
I added a ton of dungeons where not would fit before.

I will be happy to see the stand-alone build though.  :)
